practical/jython/annotate_dataset.jy

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env jython
+# -*- coding: utf-8 -*-
+#-----------------------------------------------------------------------------
+#  Copyright (C) 2017 University of Dundee. All rights reserved.
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+#------------------------------------------------------------------------------
+
+# This Jython script shows how to create and annotate a dataset.
+# The purpose of the script is to be used in the Scripting Dialog
```

Comment:
``The purpose of the script is to be used in the Scripting Dialog...`` -> ``Use this script in the Scripting Dialog...``